src/index.tsx

```diff
@@ -137,7 +139,13 @@ const ReactInputVerificationCode = ({
   }, []);
 
   React.useEffect(() => {
-    onChange(value.join(''));
+    const stringValue = value.join('')
+
+    onChange(stringValue);
+
+    if (!stringValue.includes(placeholder)) {
+      onCompleted(stringValue)
+    }
```

Comment:
```suggestion
    const stringValue = value.join('');
    const isCompleted = !stringValue.includes(placeholder);

    if (isCompleted) onCompleted(stringValue);
    onChange(stringValue);
```

A little bit of code cleaning to match the current codebase and we are good to go 🙏